prowler/providers/common/models.py

```diff
@@ -55,3 +56,16 @@ def __init__(self, arguments, bulk_checks_metadata):
             if not isdir(arguments.output_directory + "/compliance"):
                 if arguments.output_formats:
                     makedirs(arguments.output_directory + "/compliance", exist_ok=True)
+
+
+@dataclass
+class Connection:
```

Comment:
This should not be needed if you merge from `master`.

Reply:
Done